valuetransfer/src/main/java/nl/tudelft/trustchain/valuetransfer/ui/exchange/ExchangeFragment.kt.orig

```diff
@@ -0,0 +1,358 @@
+package nl.tudelft.trustchain.valuetransfer.ui.exchange
```

Comment:
What is the purpose of this file? It even seems to contains merge conflicts...

Reply:
> `.orig`

Appears to be some sort of backup.

Reply:
Removed.